addons/website_event/views/event_templates_page.xml

```diff
@@ -12,12 +12,14 @@
             <t t-set="editor_sub_message">Following content will appear on all events.</t>
             <div class="oe_structure oe_empty" id="oe_structure_website_event_layout_1" t-att-data-editor-sub-message="editor_sub_message"/>
         </div>
+
+        <!-- The registration modal is available in any event page  -->
+        <t t-call="website_event.modal_ticket_registration"/>
```

Comment:
Isn't it technically only needed when "is_rendering_cta" is True here below?

But maybe it's better to include it below the wrapwrap specifically, not entirely sure.

Reply:
Actually no, as the only time "is_rendering_cta" is False, it's for the /register page, that adds the button manually and also has this modal, so it is on all pages.

So we're good ^^